test/functional/test_site/testAnchorGeneration.md

```diff
@@ -0,0 +1,34 @@
+<frontmatter>
+  title: "Anchor Generation Test"
+</frontmatter>
+
+# Root file
+
+# should have anchor
+## should have anchor
+### should have anchor
+#### should have anchor
+##### should have anchor
+###### should have anchor
+
+<panel header="#### should have anchor">
+Lorem ipsum
+</panel>
+
+
+<panel header="Not expanded">
+    
+### Header inside panel (non-expanded)
+Should **not** have anchor
+</panel>
+
+
+<panel header="Expanded" expanded>
+    
+### Header inside panel (expanded)
```

Comment:
Should test all 6 levels of headings within a panel as well. Likewise for the included file.